scripts/sanityCheckEnLyaml.js

```diff
@@ -0,0 +1,20 @@
+const fetch = require('node-fetch');
+const fs = require('fs');
+
+const packageLockJSON = require('../package-lock.json');
+const cliLibVersion = packageLockJSON.dependencies['@hubspot/cli-lib'].version;
+
+console.log('Running precommit check for lang files...');
+(async () => {
+  const res = await fetch(
+    `https://raw.githubusercontent.com/HubSpot/hubspot-cli/v${cliLibVersion}/packages/cli-lib/lang/en.lyaml`
+  );
+  const txt = await res.text();
+  const localLyaml = fs.readFileSync('./src/lang/en.lyaml', 'utf8');
+  if (txt !== localLyaml) {
+    console.error('Local lyaml needs to be updated, replacing...');
```

Comment:
Can we update this message to include the reason why we need to update? i.e "Local lyaml out of sync with remote, updating...", "Local lyaml needs to be updated  because it is out of sync, replacing...".